README.md

```diff
@@ -1,25 +1,26 @@
 # Raspagem IoT
 
-Exemplos simples de aplica��es de raspagem de dados utilizando a placa Intel Edison.
+Exemplos simples de aplicações de raspagem de dados utilizando a placa Intel Edison.
 
-## Pr�-requisitos
+## Pré-requisitos
 - [Python 2.7](https://www.python.org/downloads/release/python-2713/)
 - [BeautifulSoup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-beautiful-soup)
 - [requests](https://github.com/requests/requests)
 - [upm](https://github.com/intel-iot-devkit/upm/blob/master/docs/installing.md)
 - [Kit Intel Edison para Arduino](https://www.intel.com.br/content/www/br/pt/products/boards-kits/edison/kit-for-arduino.html)
 - [Grove - Starter Kit Plus Gen 2](https://www.seeedstudio.com/Grove-starter-kit-plus-Intel-IoT-Edition-for-Intel-Galileo-Gen-2-and-Edison-p-1978.html)
 
-## Instala��o
+## Instalação
 
 Acesse o Edison
 
 ```
-# Substitua edison-lmayra pelo nome da sua placa
-$ ssh root@edison-lmayra.local
+# Substitua 'edison-nomedasuaplaca' pelo nome da sua placa
+$ ssh root@edison-nomedasuaplaca.local
 ```
+>É importante notar que para acessar sua placa através do SSH é necessário que a mesma esteja conectada a uma rede WiFi!
```

Comment:
Alguns pontos:

- Esse `>` no início da frase não deveria existir. Isso transformaria a sua frase em um quote (exemplo abaixo):
> exemplo

- Uso da expressão `a mesma` para fazer uma referência a algo dito previamente [é errado](http://brasilescola.uol.com.br/gramatica/utilizar-mesmo-mesmo-correto.htm) (embora seja um erro bastante comum). - pedante, eu sei! 😝  Mas acho importante o cuidado!

- Não precisa necessariamente ser WiFi. Precisa estar conectada à internet, poderia ser cabeada tbm (ethernet)

Reply:
eu achei que ficaria legal como quote, mas realmente, perde um pouco o sentido
Perdão pelo erro, é bem importante manter este cuidado sim! 

vou atualizar quanto a conexão